Library/Homebrew/test/utils/repology_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "utils/repology"
+
+describe Repology do
+  describe "formula_data", :integration_test do
```

Comment:
Ideally this would not be an integration test. Are you sure that's required?

Reply:
@MikeMcQuaid The reason for the `:integration_test` is because it seems to be required in order to use `install_test_formula "testball"`, which we are using as our test formula for some of these tests.  I tried to model this off of some of the other existing code seen in files like `formula_info_spec.rb`.  

Is there a better solution?  Thanks so much for your feedback!

Reply:
@MikeMcQuaid is this the best way to do it?
```ruby
    let(:f) { formula { url "foo-1.0" } }
    let(:options) { FormulaInstaller.new(f).display_options(f) }
    let(:action)  { "#{f.full_name} #{options}".strip }
```

Reply:
Yup, something like that would work. You can specify anything using `formula { }` .

Reply:
Great! Thanks for the feedback